File: Baekjoon/Math/숫자의 개수(2577).py
import sys
input = sys.stdin.readline

A = int(input())
B = int(input())
C = int(input())

multiple = A * B * C

result = [0]*10
while multiple:
    digit = multiple % 10
    multiple = multiple // 10

    result[digit] += 1

print('\n'.join(map(str, result)))

File: Baekjoon/Math/숫자의 개수(2577).js
/**
 * 
 */


const rl = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout,
});

const input = [];

rl.on('line', (line) => {
    input.push(line);
});

rl.on('close', () => {
    let multiple = input.reduce((a, b) => a * b, 1);
    
    const result = Array.from({ length: 10}, () => 0);
    while (multiple) {
        const digit = multiple % 10;
        multiple = (multiple / 10) |0;

        result[digit]++;
    }

    console.log(result.join('\n'));
});